src/Main.java

```diff
@@ -21,19 +21,20 @@ public static void main(String[] args) {
             System.out.println("Установите облегченную версию приложения для Android по ссылке");
         }
         System.out.println("Задача 3");
-        int year = 2024;
-        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
+        int year = 1500;
+        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0 && year>1584) {
             System.out.println(year + " год является високосным");
-        } else {
+        } else if (year%4!=0 && year %100 ==0 || year% 400 !=0 && year>1584) {
```

Comment:
зачем здесь else if? просто else должен быть, потому что это ИНАЧЕ. Всего 2 варианта - високосный и нет. Никаких дополнительных условий быть не может
```suggestion
        } else {
```